dist/index.js

```diff
@@ -4698,6 +4726,12 @@ async function maybeInstallElixir(elixirSpec, otpVersion) {
     return true
   }
 
+  if (shouldMixHex) {
+    console.log(
+      "hex will not be installed (overriding default) since Elixir wasn't either",
```

Comment:
ah, I guess this is related to https://github.com/erlef/setup-beam/pull/49#discussion_r656447109?

so someone set `install-hex: true` but left `elixir-version` out? to me that should be an error: "cannot set install-hex: true without specifying elixir-version" or something.

Reply:
Agreed, makes no sense to have one without the other.

Reply:
> so someone set install-hex: true but left elixir-version out?

The fact is it was a "no-choice". `install-hex` is `true` by default (I think it's been like this for a while - didn't check). I can:
1. improve the warning/error messages, or
2. make it default in the code, not as an input.

Whatever choice I agree this could do with some tweaking for clarity. Let me open a new issue.

Reply:
https://github.com/erlef/setup-beam/issues/50

Reply:
ohhhh, install-hex is true by default, that makes sense. in that case i would honestly silently ignore this and call it a day.